install/scripts/base-archlinux-rolling.def

```diff
@@ -2,23 +2,16 @@ BootStrap: docker
 From: archlinux/base
 
 %setup
-cp ./install-rocm.sh ${SINGULARITY_ROOTFS}/install-rocm.sh
-cp ./install-cuda.sh ${SINGULARITY_ROOTFS}/install-cuda.sh 
 cp ./install-hipsycl.sh ${SINGULARITY_ROOTFS}/install-hipsycl.sh
-cp ./install-llvm.sh ${SINGULARITY_ROOTFS}/install-llvm.sh
-cp ./add-repo-arch.sh ${SINGULARITY_ROOTFS}/add-repo-arch.sh
+cp ./install-cuda.sh ${SINGULARITY_ROOTFS}/install-cuda.sh
+cp ./spack-install-hipsycl.sh  ${SINGULARITY_ROOTFS}/spack-install-hipsycl.sh 
+cp ./spack-install-llvm.sh ${SINGULARITY_ROOTFS}/spack-install-llvm.sh 
+cp ./spack-install-boost.sh ${SINGULARITY_ROOTFS}/spack-install-boost.sh 
+cp ./spack-install-rocm.sh ${SINGULARITY_ROOTFS}/spack-install-rocm.sh 
+cp ./spack-syclcc.json ${SINGULARITY_ROOTFS}/syclcc.json
```

Comment:
Why do we need to copy `syclcc.json` in? Shouldn't it be possible to let cmake generate it and configure cmake so that it generates it as we require?

Reply:
I tried to do that but my cmake knowledge has proved to be insufficient for this task. The problem is that `find_package(HIP)` doesn't set the variables that we need but only imports the `hip::host` and `hip::device` targets. I tried to retrieve the paths using `get_properties` but the dependencies also only use other imported targets as dependencies not actual paths. I would be very happy to see if there is a clean solution for that :)